### DISCUSSION TITLE
Is it possible to prefetch a url programmatically?

### DISCUSSION THREAD

**Discussion by Saad5400** [livewire/livewire#8825] [1. Help]
NOT on hover, but programmatically. For example in my use case I have a quiz, which has multiple questions. Each question has it's own url and page, with a 'Next' button. I want to load that next question before the user even clicks on the next button, just to make the experience better for the user.

It's possible on Svelte with a function called 'preloadData', wondering if I can do the same here?

**Comment by mityukov** [comment]
Affects me too. In particular, there's no "hover" event in "touch" devices, so, I'd like to have a way to prefetch sibling page just after the current page is loaded (or, maybe, on some timeout -- to be decided later).

**Reply by Saad5400** [reply]
I switched to Inertia as it supports this feature: https://inertiajs.com/prefetching
